glambie/entrypoint_for_debugging.py

```diff
@@ -0,0 +1,21 @@
+"""
```

Comment:
This feels like the wrong way to achieve this. I.e. it doesn't belong here as it's not necessary for core functionality of the package (especially at the root level). I think I would add this functionality into entrypoint.py with something like.

# This exists for the purposes of interactive debugging
function _ide_debug(config_path):
    Current logic

# This exists for the purposes of interactive debugging
if __name__ == '__main__':
    _ide_debug("configuration/0_parent_config.yaml")

Or even better, can main be called directly after some initial setup in "if __name__ == '__main__':"?